app/assets/stylesheets/modules/_global-footer.scss

```diff
@@ -17,3 +17,14 @@
     @include underline;
   }
 }
+
+.color-palette {
+
+  @include media($medium-screen-up) {
+  background-image: url("color-palette-footer.png");
+  background-repeat: no-repeat;
+  height: 40px;
+  width: 25%;
```

Comment:
Line should be indented 4 spaces, but was indented 2 spaces<br>Properties should be ordered background-image, background-repeat, height, margin, width